test/integration/webhook/leaderworkerset_test.go

```diff
@@ -12,7 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package webhook
```

Comment:
nit: still webhooks? We will not import this package. No name conflicts.

Reply:
Rename the folder to webhooks.

Reply:
done.